content/intro/getting-started/index.md

```diff
@@ -8,4 +8,12 @@ weight: 300
 toc: true
 resources:
   - src:
----
\ No newline at end of file
+---
+
+Signing up for a BluBracket account takes only a few minutes. We aim to help you find the risks in your code, improve code security, and dig deeper into the information provided by our scanning tool.
```

Comment:
Scope clarification: we want to get the user all the way from clicking "start now" on https://blubracket.com through to having their first code server configured.